client/pages/index.tsx

```diff
@@ -22,13 +23,17 @@ const pageStates = {
   RESULTS: 'RESULTS', // displays results
   API_ERROR: 'API_ERROR', // API error page
```

Comment:
feel free to remove the API_ERROR/ERROR -- this was kinda like a guide but it's not used anymore haha